arrow-core/src/main/kotlin/arrow/core/extensions/const/apply/ConstApply.kt

```diff
@@ -100,8 +100,8 @@ fun <A, B, Z> Kind<Kind<ForConst, A>, A>.map2Eval(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "map(MA, arg0, arg1, arg2)",
-  "arrow.core.Const.map"
+    "Cons.mapN(MA, arg0, arg1, arg2)",
+    "arrow.core.Cons"
```

Comment:
```suggestion
    "Cons.mapN(MA, arg0, arg1) { a, b -> arg2(Tuple2(a, b)) } ",
    "arrow.core.Cons", "arrow.core.Tuple2"
```

The deprecations for `mapN` don't match the correct signature.